autopush/router/simple.py

```diff
@@ -76,6 +75,9 @@ def route_notification(self, notification, uaid_data):
         # Determine if they're connected at the moment
         node_id = uaid_data.get("node_id")
         uaid = uaid_data["uaid"]
+        self.udp = None
+        if "udp" in uaid_data:
+            self.udp = uaid_data["udp"]
```

Comment:
Nope, if "udp" is in the dict, you'll never get a key error. Technically the way you have it here is marginally faster than using dict.get as well, because get() invokes more of the function call overhead while the "udp" in uaid_data is a pure key check at a lower level without Python function overhead setup. So either way is fine by me. :)